app/controllers/Round.scala

```diff
@@ -137,8 +137,19 @@ object Round extends LilaController with TheftPrevention {
   def watcher(gameId: String, color: String) = Open { implicit ctx =>
     GameRepo.pov(gameId, color) flatMap {
       case Some(pov) =>
-        env.checkOutoftime(pov.game)
-        watch(pov)
+        get("pov") match {
+          case Some(requestedPov) => (pov.player.userId, pov.opponent.userId) match {
+            case (Some(_),Some(opponent)) if opponent == requestedPov =>
+              Redirect(routes.Round.watcher(gameId, (!pov.color).name)).fuccess
+            case (Some(player),Some(_)) if player == requestedPov =>
+              Redirect(routes.Round.watcher(gameId, pov.color.name)).fuccess
+            case _ => notFound
```

Comment:
actually I'd prefer if it always redirects.

E.g. `/abcdefgh?pov=thibault` redirects to `/abcdefgh/black`.

A resource should not have several urls.